modules/fundamental/src/advisory/model/details/advisory_vulnerability.rs

```diff
@@ -59,15 +59,13 @@ impl AdvisoryVulnerabilityHead {
 
         if let Some(advisory_vuln) = advisory_vuln {
             Ok(AdvisoryVulnerabilityHead {
-                head: VulnerabilityHead::from_vulnerability_entity(vulnerability, tx).await?,
+                head: VulnerabilityHead::from_vulnerability_entity(vulnerability),
```

Comment:
I think `tx` might be un-used currently, but was keeping the same pattern so that in the future (which may never arrive) it'd be available, and async, to allow for further scrobbling around in the DB to populate iff needed.

Possibly YAGNI. Not a strongly-held opinion.

Reply:
Yea, I saw that. Same as the fn being async. However, removing that I noticed that some code could be re-used.